packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -20,8 +20,6 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   'about-dialog.version-info.auto-updates.how-to-enable': 'Enable',
   /** "Manage version" link text */
   'about-dialog.version-info.auto-updates.manage-version': 'Manage version',
-  /** "Browse on GitHub" link from version info dialog */
-  'about-dialog.version-info.browse-on-github': 'Browse on GitHub',
```

Comment:
I was gonna call out the key rename but it seems like we're not using it yet anyway: https://github.com/search?q=repo%3Asanity-io%2Flocales%20about-dialog.version-info.browse-on-github&type=code